nvflare/app_common/storages/filesystem_storage.py

```diff
@@ -162,7 +176,7 @@ def clone_object(self, from_uri: str, to_uri: str, meta: dict, overwrite_existin
 
         from_full_uri = self._object_path(from_uri)
         from_data_path = os.path.join(from_full_uri, DATA)
-        _write(data_path, from_data_path)
+        _write(data_path, from_data_path, mv_file=False)
```

Comment:
Line 158 is fine since the file is supposed to be "moved". In old version, the file was always copied, but could be slow when the file is very big (e.g. containing weights).